### PR TITLE
DHFPROD-8226: Results-tabular-view - Expandable

### PR DESCRIPTION
**Pull request by iveyMarklogic**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/results-tabular-view/results-tabular-view.tsx [hunk 1]
```diff
@@ -360,19 +361,44 @@ const ResultsTabularView = (props) => {
...
 
   const expandedRowRender = (rowId) => {
     const nestedColumns = [
-      {title: "Property", dataIndex: "property", width: "33%"},
-      {title: "Value", dataIndex: "value", width: "34%"},
-      {title: "View", dataIndex: "view", width: "33%"},
+      {
+        text: "Property",
+        title: "Property",
+        dataIndex: "property",
```
Comment (mrodrigu-marklogic) [inline marklogic-data-hub-central/ui/src/components/results-tabular-view/results-tabular-view.tsx:373]: These properties should be changed from `dataIndex` > `dataField`

**Review by mrodrigu-marklogic — CHANGES_REQUESTED** [review]

File: marklogic-data-hub-central/ui/src/components/results-tabular-view/results-tabular-view.tsx (hunk 1 above)
Comment (iveyMarklogic) [reply marklogic-data-hub-central/ui/src/components/results-tabular-view/results-tabular-view.tsx]: @mrodrigu-marklogic can you check again

**Review by MSBrignoni — DISMISSED** [review]

**Review by yubiselmarklogic — APPROVED** [review]

**Review by Sanjeevani19 — COMMENTED** [review]
@iveyMarklogic Looks good, Please rebase your PR

**Review by Sanjeevani19 — DISMISSED** [review]

**Review by briantang — APPROVED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Review by mrodrigu-marklogic — APPROVED** [review]

**Review by Sanjeevani19 — CHANGES_REQUESTED** [review]
@iveyMarklogic Lint issues needs to be fixed on this PR

**Review by mrodrigu-marklogic — APPROVED** [review]

**Review by yubiselmarklogic — APPROVED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Pull request merged**